pkg/proxy/util/utils.go

```diff
@@ -477,6 +477,14 @@ func WriteLine(buf *bytes.Buffer, words ...string) {
 	}
 }
 
+// WriteRuleLine joins all words with spaces, terminate with newline and write to buff.
+func WriteRuleLine(buf *bytes.Buffer, chainName string, words ...string) {
+	if len(chainName) <= 0 || len(words) <= 0 {
+		return
+	}
+	WriteLine(buf, append([]string{"-A", chainName}, words...)...)
```

Comment:
I think this was how I had suggested implementing it before, but seeing the comment in `WriteLine` about "performance reasons" makes me feel guilty about adding an extra append here. You could instead do:

    buf.WriteString("-A ")
    buf.WriteString(string(chainName))
    WriteLine(buf, words...)

Reply:
ack.